pkg/registry/core/secret/registry.go

```diff
@@ -67,12 +67,12 @@ func (s *storage) GetSecret(ctx genericapirequest.Context, name string, options
 }
 
 func (s *storage) CreateSecret(ctx genericapirequest.Context, secret *api.Secret) (*api.Secret, error) {
```

Comment:
comment that its only safe due to usage and that we need to move to a loopback connection for SA token authentication.

Reply:
issue for the loopback thing.

Reply:
Also dead code fixed via #54693